python/doc/examples/reliability_sensitivity/design_of_experiments/plot_smolyak_indices.py

```diff
@@ -33,10 +33,10 @@
 # In the first example, we print the indices involved in
 # Smolyak-Legendre quadrature of level 3.
 # The multi-indices are computed using the
-# :meth:`~openturns.SmolyakExperiment.computeCombination` method.
+# :py:meth:`openturns.experimental.SmolyakExperiment.computeCombination` method.
```

Comment:
Why is "py" necessary (here and at other places)?

Reply:
Maybe its the same, the problem was really the missing "experimental"